src/olympia/files/tests/test_utils_.py

```diff
@@ -512,64 +512,73 @@ def test_extract_translations_fail_silent_missing_file(read_mock, file_obj):
 
-def test_resolve_i18n_message_uses_default_locale():
-    messages = {
-        'en-US': {
-            'foo': {'message': 'bar'}
+    def test_uses_default_locale(self):
+        messages = {
+            'en-US': {
+                'foo': {'message': 'bar'}
+            }
         }
-    }
-
-    result = utils.resolve_i18n_message('__MSG_foo__', messages, 'de', 'en')
-    assert result == 'bar'
 
+        result = utils.resolve_i18n_message(
+            '__MSG_foo__', messages, 'de', 'en')
+        assert result == 'bar'
 
-def test_resolve_i18n_message_no_locale_match():
-    # Neither `locale` or `locale` are found, "message" is returned unchanged
-    messages = {
-        'fr': {
-            'foo': {'message': 'bar'}
+    def test_no_locale_match(self):
+        # Neither `locale` or `locale` are found, "message" is returned
+        # unchanged
+        messages = {
+            'fr': {
+                'foo': {'message': 'bar'}
+            }
         }
-    }
 
-    result = utils.resolve_i18n_message('__MSG_foo__', messages, 'de', 'en')
-    assert result == '__MSG_foo__'
+        result = utils.resolve_i18n_message(
+            '__MSG_foo__', messages, 'de', 'en')
+        assert result == '__MSG_foo__'
 
+    def test_field_not_set(self):
+        """Make sure we don't fail on messages that are `None`
 
-def test_resolve_i18n_message_field_not_set():
-    """Make sure we don't fail on messages that are `None`
-
-    Fixes https://github.com/mozilla/addons-server/issues/3067
-    """
-    result = utils.resolve_i18n_message(None, {}, 'de', 'en')
-    assert result is None
-
-
-def test_resolve_i18n_message_field_no_string():
-    """Make sure we don't fail on messages that are no strings"""
-    result = utils.resolve_i18n_message([], {}, 'de', 'en')
-    assert result == []
+        Fixes https://github.com/mozilla/addons-server/issues/3067
+        """
+        result = utils.resolve_i18n_message(None, {}, 'de', 'en')
+        assert result is None
+
+    def test_field_no_string(self):
+        """Make sure we don't fail on messages that are no strings"""
+        result = utils.resolve_i18n_message([], {}, 'de', 'en')
+        assert result == []
+
+    def test_corrects_locales(self):
+        messages = {
+            'en-US': {
+                'foo': {'message': 'bar'}
+            }
+        }
 
+        result = utils.resolve_i18n_message('__MSG_foo__', messages, 'en')
+        assert result == 'bar'
 
-def test_resolve_i18n_message_corrects_locales():
-    messages = {
-        'en-US': {
-            'foo': {'message': 'bar'}
+    def test_ignore_wrong_format(self):
+        messages = {
+            'en-US': {
+                'foo': 'bar'
+            }
         }
-    }
 
-    result = utils.resolve_i18n_message('__MSG_foo__', messages, 'en')
-    assert result == 'bar'
+        result = utils.resolve_i18n_message('__MSG_foo__', messages, 'en')
+        assert result == '__MSG_foo__'
```

Comment:
the email may look like that but it's not supposed to end up in the comment thread on github, it's supposed to be stripped out.